src/explorer/README.md

```diff
@@ -16,6 +16,11 @@ This app runs as a docker container. To build the image `docker-compose build ex
 
 In the project directory, you can run:
 
+#### `npm run build:full-css`
+
+- creates `src/styles/tailwind.css`
+- do this before running `npm start` for the first time
+
```

Comment:
**question**: should this be a npm start hook or something?

Reply:
Probably. I haven't yet researched the two different build css tasks and wasn't really clear on the difference between them where.